docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -4,7 +4,7 @@ source:
   ref: batchSize
 ---
 source:
-  file: apiargs-common-option.yaml
+  file: apiargs-MongoDBCollection-common-option.yaml
```

Comment:
Since this is the collection helper, it should inherit any collation on the collection. This was already mentioned in the docs for `countDocuments()`, but I expect it was missed here due to the `watch()` docs being written for Client, Database, and Collection concurrently.